shared/src/proxies/documents/batchDownloadDocketEntriesProxy.ts

```diff
@@ -0,0 +1,13 @@
+import { DownloadDocketEntryRequestType } from '@shared/business/useCases/document/batchDownloadDocketEntriesInteractor';
+import { get } from '../requests';
+
+export const batchDownloadDocketEntriesInteractor = (
+  applicationContext,
+  params: DownloadDocketEntryRequestType,
+) => {
+  return get({
+    applicationContext,
+    endpoint: '/async/documents/batch-download',
```

Comment:
Should this endpoint specify it's downloading docket entries?

Reply:
We are specifically downloading the documents "associated" with the docket entries. We aren't downloading the docket entries's data itself.